packages/shared/src/modules/users/libs/validation-schemas/user-reset-password.validation-schema.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from "zod";
+
+import { UserValidationMessage, UserValidationRule } from "../enums/enums.js";
+
+type UserSignInRequestValidationDto = {
+	confirmPassword: z.ZodString;
+	newPassword: z.ZodString;
+};
+
+const userResetPassword = z
+	.object<UserSignInRequestValidationDto>({
+		confirmPassword: z
```

Comment:
Shouldn't be sent to Backend

Reply:
So I should move this schema to frontend?